Android/Samples/libraries/geojson/build.gradle

```diff
@@ -1,6 +1,5 @@
 apply plugin: 'com.android.library'
 apply plugin: 'maven'
-apply plugin: "com.github.sherter.google-java-format"
```

Comment:
Don't you need this to format code?

Reply:
It is in the top-level build.gradle for the entire project, so it was redundant in the build.gradle for the module.